### PR TITLE
Added a new question about git (git fetch command)

### PR DESCRIPTION
**Pull request by sotoblanco**
Question that allow to download objects without having to commit to the main branch. A great feature to consider when you are working on projects using different sources.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/git-quiz.ts [hunk 1]
```diff
@@ -586,6 +586,17 @@ const gitQuiz = [
     Explanation:
       "git commit -am  '<commit message>' is used to stage and commit all changes in the current directory and its subdirectories.",
     Link: "https://git-scm.com/docs/git-commit"
+  },
+    {
+    Question:
+      "What git command allow to download objects and refs from another repository without commiting to the main branch?",
```
Comment (jdwilkin4) [inline src/data/git-quiz.ts:592]:
```suggestion
      "Which git command allows you to download objects and refs from another repository without committing to the main branch?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by sotoblanco — COMMENTED** [review]
Thanks!

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by sotoblanco** [comment]
Thank you so much for your support!!

**Pull request merged**